Helper_Scripts/TTS_Installers/install_tts_supertonic.py

```
@@ -0,0 +1,160 @@
+    base_dir = Path(args.base_dir).expanduser()
+    onnx_dir = (base_dir / args.onnx_dir).expanduser()
+    voice_dir = (base_dir / args.voice_styles_dir).expanduser()
+
+    clone_root = base_dir / "_supertonic_clone" if args.keep_clone else Path(tempfile.mkdtemp(prefix="supertonic_clone_"))
+    if clone_root.exists():
+        if any(clone_root.iterdir()):
+            raise SystemExit(f"Clone directory already exists and is not empty: {clone_root}")
+    clone_root.parent.mkdir(parents=True, exist_ok=True)
+
+    try:
+        print(f"Installing git lfs filters...")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)